### PR TITLE
OB arithmetic function test refactor

### PR DESCRIPTION
**Pull request by Burdishka**
Refactoring user story: https://trello.com/c/klAEa9DY/322-entityarithmeticfunctiontest-clickarithmeticfunction

### CODE REVIEW

**Review by guram21 — APPROVED** [review]

File: src/test/java/EntityArithmeticFunctionTest.java [hunk 1]
```diff
@@ -3,228 +3,175 @@
...
 
-    public void clickSaveButton(WebDriver driver) {
+    private void valuesAssertViewMode(WebDriver driver, int number1, int number2) {
 
-        WebElement saveButton = driver.findElement(By.id("pa-entity-form-save-btn"));
-        saveButton.click();
+        Assert.assertEquals(String.valueOf(number1 + number2), driver.findElement(SUM_VIEW_MODE).getText());
+        Assert.assertEquals(String.valueOf(number1 - number2), driver.findElement(SUB_VIEW_MODE).getText());
+        Assert.assertEquals(String.valueOf(number1 * number2), driver.findElement(MUL_VIEW_MODE).getText());
+        Assert.assertEquals(String.valueOf(number1 / number2), driver.findElement(DIV_VIEW_MODE).getText());
```
Comment (Stan575) [inline src/test/java/EntityArithmeticFunctionTest.java:87]: Please change arguments oder for assertEquals in these lines 
`assertEquals(String actual,String expected)`

File: src/test/java/EntityArithmeticFunctionTest.java [hunk 2]
```diff
@@ -3,228 +3,175 @@
...
-
-        WebElement mul = driver.findElement(By.xpath("//div[5]//span"));
-        Assert.assertEquals(mul.getText(), String.valueOf(f1 * f2));
-
-        WebElement div = driver.findElement(By.xpath("//div[6]//span"));
-        Assert.assertEquals(div.getText(), String.valueOf(f1 / f2));
+        Assert.assertEquals(String.valueOf(number1 + number2), driver.findElement(SUM_EDIT_MODE).getAttribute("value"));
+        Assert.assertEquals(String.valueOf(number1 - number2), driver.findElement(SUB_EDIT_MODE).getAttribute("value"));
+        Assert.assertEquals(String.valueOf(number1 * number2), driver.findElement(MUL_EDIT_MODE).getAttribute("value"));
+        Assert.assertEquals(String.valueOf(number1 / number2), driver.findElement(DIV_EDIT_MODE).getAttribute("value"));
```
Comment (Stan575) [inline src/test/java/EntityArithmeticFunctionTest.java:95]: Change arguments oder

File: src/test/java/EntityArithmeticFunctionTest.java [hunk 3]
```diff
@@ -3,228 +3,175 @@
...
-        clickEditMode(driver);
+        ProjectUtils.click(driver, driver.findElement(EDIT_MODE));
 
-        final int f1 = 24;
-        final int f2 = 8;
+        setValuesF1F2String(driver, STRING, STRING);
+        driver.findElement(SAVE_BUTTON).click();
 
-        valuesAssertEditMode(driver, f1, f2);
+        Assert.assertEquals(ERROR_MESSAGE, driver.findElement(ERROR).getText());
```
Comment (Stan575) [inline src/test/java/EntityArithmeticFunctionTest.java:161]: Change arguments oder

**Review by Stan575 — CHANGES_REQUESTED** [review]
Overall looks pretty good
Minor changes requested

**Review by Stan575 — APPROVED** [review]
Well done!

**Pull request merged**